doc/gulpfile.js

```diff
@@ -1,59 +1,84 @@
-var gulp        = require('gulp'),
-	autoprefixer= require('gulp-autoprefixer'),
-	livereload  = require('gulp-livereload'),
-	notify      = require('gulp-notify'),
-	sass        = require('gulp-sass'),
-	gulpif      = require('gulp-if'),
-	sassLint    = require('gulp-sass-lint'),
-	sourcemaps  = require('gulp-sourcemaps');
+var gulp = require('gulp'),
+    autoprefixer = require('gulp-autoprefixer'),
+    livereload = require('gulp-livereload'),
+    notify = require('gulp-notify'),
+    sass = require('gulp-sass'),
+    gulpif = require('gulp-if'),
+    sassLint = require('gulp-sass-lint'),
+    serve = require('gulp-serve'),
+    sourcemaps = require('gulp-sourcemaps'),
+    exec = require('child_process').exec;
```

Comment:
You replaced tabs for spaces!
Your editor should honnor the `.editorconfig` !

Reply:
Just converted the gulpfile indentation back to tabs. Sorry about that.